tokio-console/src/view/task.rs

```diff
@@ -127,16 +138,30 @@ impl TaskView {
             )
         };
 
-        let stats_area = Layout::default()
-            .direction(layout::Direction::Horizontal)
-            .constraints(
-                [
-                    layout::Constraint::Percentage(50),
-                    layout::Constraint::Percentage(50),
-                ]
-                .as_ref(),
-            )
-            .split(stats_area);
+        let stats_area = if location_lines_vector.len() != 1 {
+            let area_needed_to_render_location = task.location().len() as u16;
+            Layout::default()
+                .direction(layout::Direction::Horizontal)
+                .constraints(
+                    [
+                        layout::Constraint::Min(area_needed_to_render_location + 15), //Note: 15 is the length of "| Location:     |"
+                        layout::Constraint::Min(32),
+                    ]
+                    .as_ref(),
+                )
+                .split(stats_area)
+        } else {
+            Layout::default()
+                .direction(layout::Direction::Horizontal)
+                .constraints(
+                    [
+                        layout::Constraint::Percentage(50),
+                        layout::Constraint::Percentage(50),
+                    ]
+                    .as_ref(),
+                )
+                .split(stats_area)
+        };
```

Comment:
We can simplify this a bit by just setting different constraints:

```suggestion
        let stats_constraints = if location_lines_vector.len() > 1 {
            [
                // 15 is the length of "| Location:     |"
                layout::Constraint::Min(area_needed_to_render_location + 15),
                layout::Constraint::Min(32),
            ]
        } else {
            [
                layout::Constraint::Percentage(50),
                layout::Constraint::Percentage(50),
            ]
        };
        let stats_area = Layout::default()
            .direction(layout::Direction::Horizontal)
            .constraints(stats_constraints.as_ref())
            .split(stats_area);
```